WordPress/Classes/Models/Notifications/Notification.swift

```diff
@@ -81,10 +81,29 @@ class Notification: NSManagedObject {
     ///
     fileprivate var cachedHeaderAndBodyContentGroups: [FormattableContentGroup]?
 
+    private let cachedAttributesObserver: NotificationCachedAttributesObserver? = nil
+
     /// Array that contains the Cached Property Names
     ///
     fileprivate static let cachedAttributes = Set(arrayLiteral: "body", "header", "subject", "timestamp")
 
+    override init(entity: NSEntityDescription, insertInto context: NSManagedObjectContext?) {
+        super.init(entity: entity, insertInto: context)
+
+        cachedAttributesObserver = NotificationCachedAttributesObserver()
+        for attr in Notification.cachedAttributes {
+            addObserver(cachedAttributesObserver!, forKeyPath: attr, context: nil)
+        }
```

Comment:
What do you think of doing the following, to remove the force unwrap?

```suggestion
        let cachedAttributesObserver = NotificationCachedAttributesObserver()
        for attr in Notification.cachedAttributes {
            addObserver(cachedAttributesObserver, forKeyPath: attr, context: nil)
        }
        self.cachedAttributesObserver = cachedAttributesObserver
```

The force unwrap is of course okay in the way it's used now, but who's to tell how the code will evolve? There might end up being a code path in which it's `nil` in the future.

Maybe I'm overthinking it 😅

Reply:
👍 addressed in https://github.com/wordpress-mobile/WordPress-iOS/pull/19395/commits/da753bdff58fa9e39efd00c9e758efc50fc308ef. It's always good to not use force unwrap.